recipe/meta.yaml

```
@@ -81,7 +81,7 @@ outputs:
       run_constrained:
         - napari-plugin-manager >=0.1.0a2,<0.3.0a0
         - pyside2 >=5.13.2,!=5.15.0
-        - pyside6 <6.5,>=6.7
+        - pyside6 !=6.6.*
```

```suggestion
        - pyside6 <6.5|>=6.7
```

this syntax might work too.

---

actually this is the correct syntax since you are trying to avoid all 6.5 and 6.6.

---

Oops, sorry, yes, we needed a `|` not `,`. Thanks @hmaarrfk!